components/raftstore-v2/src/operation/command/control.rs

```diff
@@ -82,7 +82,10 @@ pub struct ProposalControl {
     // should be empty or 1 element. And access speed is not a concern.
     proposed_admin_cmd: LinkedList<ProposedAdminCmd>,
     has_pending_prepare_merge: bool,
-    applied_prepare_merge_index: u64,
+    // Commit index of prepare merge.
+    // None means there is no prepare merge.
+    // Some(0) means there is a prepare merge, and it has applied.
+    applied_prepare_merge_index: Option<u64>,
```

Comment:
This maybe is not necessary. Because merge index cannot be 0 anyway. So 0 means none. 
And after it's applied, then we can think it's same as if there's no merge at all. So to me the original u64 is ok and it's simpler.